operator/pkg/reconciliation/constructor.go

```diff
@@ -362,6 +362,21 @@ func probe(port int, path string, initDelay int, period int) *corev1.Probe {
 	}
 }
 
+func getJvmExtraOpts(dc *api.CassandraDatacenter) string {
+	flags := ""
+
+	if dc.Spec.DseWorkloads.AnalyticsEnabled == true {
+		flags += "-k "
+	}
+	if dc.Spec.DseWorkloads.GraphEnabled == true {
+		flags += "-g "
```

Comment:
is it possible that having trailing whitespace in the flags string would cause any issues?